civiwiki/frontend/static/views/friends_list_view.js

```diff
@@ -0,0 +1,103 @@
+var FriendsListView = Backbone.View.extend({
+    el: '#friends', 
+
+    friendsTemplate: _.template($('#friends-template').html()),
+
+    initialize: function(options){
+      var _this = this; 
+
+      options = options || {}; 
+
+      _this.user_id = options.user_id;
+      _this.friend_requests = options.friend_requests;
```

Comment:
@gracewylu I figured this out.  `options.friend_requests` is not a thing.  Passing in from options happens here: https://github.com/CiviWikiorg/civiwiki.org/blob/friends_tab/civiwiki/frontend/templates/account_home.html#L115

I see that here you are grabbing it out of the user model: https://github.com/CiviWikiorg/civiwiki.org/blob/friends_tab/civiwiki/frontend/static/views/account_base_view.js#L15

So to set up a listener what you did was great (legitimately awesome btw) but `_this.friend_requests` was wrong to start with.  Instead of `_this.friend_requests = options.friend_requests` do `_this.friend_requests = new Backbone.Collection(_this.userModel.toJSON().friend_requests)` after line 13 of this file.  This way we can listen for change events on the collection.  Great job knowing to do that in the first place btw, you are a Backbone pro!